src/ocamlorg_frontend/pages/home.eml

```diff
@@ -165,11 +165,11 @@ with an emphasis on expressiveness and safety."
               BUILD ANYTHING
             </div>
             <h3 class="text-body-600 font-bold">
-              First class editor and tooling
+              First Class Editor and Tooling
             </h3>
             <div class="my-4 text-lg">
               OCaml has several editor tools that make it easier to program in. Usually, VS Code is recommended for
-              beginners, but there is also good support for Vim and Emacs as well. Tools like the toplevel Utop and
+              beginners, but there is also good support for Vim and Emacs. Tools like the toplevel Utop and
```

Comment:
```suggestion
              beginners, but Vim and Emacs also offer good support. Tools like the toplevel UTop and
```

Reply:
This changes the meaning of the sentence subtly:
Neither editor (Vim or Emacs) support OCaml out of the box.
Instead the OCaml community offers configuration, supplementary tools, ease-of-setup, ... to make OCaml development easy in both editors. Hence "there is good support for" - but "Vim and Emacs also offer good support" is not quite true.

Reply:
I see what you mean. My apologies!
I'll revert it.